source/index.html

```diff
@@ -66,6 +77,18 @@
       </div>
     </section>
 
+		<!-- Misha/Laurence added: -->
+    <section class="section--recipe-cards">
```

Comment:
What is this for? We have sections for recipe cards within each of the sections (explore, search results, etc)

Reply:
^ im also kinda confused. we already have a section below with class="recipe-info" for the expanded recipe where we can append the ExpandedRecipePage web component as a child when we need to load it and i dont think we should need any new sections?